packages/ui-a11y-content/src/AccessibleContent/index.tsx

```diff
@@ -44,15 +44,15 @@ class AccessibleContent extends Component<
 > {
   static propTypes = {
     alt: PropTypes.string,
-    as: PropTypes.elementType,
-    children: PropTypes.node
-  }
+    children: PropTypes.node,
+    as: PropTypes.elementType
+  } as const
 
   static defaultProps = {
     alt: undefined,
     as: 'span',
     children: null
-  }
+  } as const
```

Comment:
Casting `propTypes` and `defaultProps` to `const` was the only way keep the current `defaultProps` functionality (required props being not really required when they have `defaultProps` entries) while getting `getElementType` utility work properly.

Reply:
Hm, this way it will loose the link between the propTypes and the Props type, but hopefully the combination of the refact we are doing in the allowedProps ticket will solve this. Anyway, we'll have to test it well when we merge the two.